IHP/Job/Dashboard/Utils.hs

```diff
@@ -7,10 +7,14 @@ import qualified Database.PostgreSQL.Simple.Types as PG
 
 numberOfPagesForTable :: (?modelContext::ModelContext) => Text -> Int -> IO Int
 numberOfPagesForTable table pageSize = do
-    (PG.Only totalRecords : _) <- sqlQuery
-        (PG.Query $ cs $ "SELECT COUNT(*) FROM " <> table)
-        ()
+    totalRecords <- totalRecordsForTable table
     pure $ case totalRecords `quotRem` pageSize of
         (pages, 0) -> pages
         (pages, _) -> pages + 1
 
+totalRecordsForTable :: (?modelContext::ModelContext) => Text -> IO Int
+totalRecordsForTable table = do
```

Comment:
Next to `sqlQuery` we also have [`sqlQueryScalar`](https://ihp.digitallyinduced.com/api-docs/IHP-ModelSupport.html#v:sqlQueryScalar) which only expects a single scalar result value (e.g. a single Integer like here). Using this function we could write this a bit shorter:

```haskell
totalRecordsForTable table = sqlQueryScalar (PG.Query $ cs $ "SELECT COUNT(*) FROM " <> table) ()
```